src/main/java/ComputerPlayer.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class ComputerPlayer implements Player{
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All classes and interfaces must belong to a named package](https://www.codacy.com/app/mollie-c-stephenson/TicTacToe/file/2534705881/issues/source?bid=3284885&fileBranchId=3284887#l4)